### PR TITLE
updates: Clarify that we do not support skipping major releases

### PR DESCRIPTION
**Pull request by travier**

### CODE REVIEW

**Comment by miabbott** [comment]
What's the reasoning behind this change/warning?

It should be *technically* possible to rebase from N to N+2, but I could understand why it might not be a good idea in practice.  Are we just trying to avoid extra bugs/issues filed on the tracker?

**Comment by travier** [comment]
We've had rpm-ostree bugs and changes that prevented that from working. The update from bdb to sqlite for example required latest rpm-ostree to be able to update to the next version. In general, we don't test it at all and we don't update Fedora N-1 versions of rpm-ostree that much.

**Comment by miabbott** [comment]
> The update from bdb to sqlite for example required latest rpm-ostree to be able to update to the next version. In general, we don't test it at all and we don't update Fedora N-1 versions of rpm-ostree that much.

Ah, right...that is a good example.  Thanks for the reminder/context!

File: modules/ROOT/pages/updates-upgrades-rollbacks.adoc [hunk 1]
```diff
@@ -30,25 +30,36 @@ Alternatively, to check for available updates without downloading them, run:
 Upgrading between major versions (such as from Fedora 35 to Fedora 36) can be completed using the Software application.
 Alternatively, {variant-name} can be upgraded between major versions using the `ostree` command.
 
-First, verify the branch is available.
-You can print all available branches with this command:
+NOTE: Skipping major releases is not supported on Silverblue.
```
Comment (miabbott) [inline modules/ROOT/pages/updates-upgrades-rollbacks.adoc:33]: Given the added context about lack of testing, could you add a small statement about that here?

**Review by miabbott — APPROVED** [review]

**Pull request merged**